src/partials/features-zigzag.tsx

```diff
@@ -148,11 +162,21 @@ function FeaturesZigZag() {
                     isDarkTheme ? '#e3e3e3' : '#1c1e21'
                   }]`}
                 >
-                  <CSharpNoFillSvg className="flex-auto h-20 w-20 m-4 " />
-                  <FontAwesomeIcon className="flex-auto h-20 w-20 m-4" icon={faGolang} />
-                  <FontAwesomeIcon className="flex-auto h-20 w-20 m-4" icon={faJava} />
-                  <FontAwesomeIcon className="flex-auto h-20 w-20 m-4" icon={faSquareJs} />
-                  <FontAwesomeIcon className="flex-auto h-20 w-20 m-4" icon={faPhp} />
+                  <Link target="_blank" to="https://github.com/open-feature/dotnet-sdk-contrib/tree/main/src/OpenFeature.Contrib.Providers.GOFeatureFlag">
```

Comment:
Wrapping the images in a Link element is causing the image colors to be overridden.

Light mode:
![image](https://user-images.githubusercontent.com/682996/235463623-94a9227e-1d7b-4377-b785-006df5f908e5.png)

Dark mode:
![image](https://user-images.githubusercontent.com/682996/235463659-ff397dc4-b54f-4ea4-91d2-2bc247468002.png)

I think it should also have a hover effect so the user can quickly tell that it's something a user can click.

Reply:
@beeme1mr Would you like to suggest to me any color code, that I should use on the hover effect?